swiftgen-cli/main.swift

```diff
@@ -10,8 +10,7 @@ import PathKit
 
 // MARK: Common
 
-let templatesRelativePath = "../Resources/templates"
-
+let templatesRelativePath = (Bundle.main.resourcePath ?? "") + "/templates"
```

Comment:
I'd use `?? "."` instead. Because in case `Bundle.main.resourcePath` returns `nil`, we would end up pointing to the root of the filesystem, which seems dangerous :wink: